script/strategies/compound-v3/arbitrum/DeployStrategies.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { BaseDeployStrategies, ICERC20 } from "../BaseDeployStrategies.sol";
+import { DeployPeriphery } from "script/DeployPeriphery.sol";
+
+contract DeployStrategies is DeployPeriphery, BaseDeployStrategies {
+  function run() external override(DeployPeriphery) {
+    vm.startBroadcast();
+    deployPeriphery();
+    deployAaveV3Strategies();
+    vm.stopBroadcast();
+  }
+
+  function deployAaveV3Strategies() internal {
```

Comment:
Why does it say aave? Aren't we deploying Compound?
Let's also take guardians and judges as params